lxd/db/cluster/identities.mapper.go

```diff
@@ -0,0 +1,511 @@
+  FROM identities
+  WHERE ( identities.id = ? )
+  ORDER BY identities.auth_method, identities.identifier
+`)
+
+var identityObjectsByAuthMethod = RegisterStmt(`
+SELECT identities.id, identities.auth_method, identities.type, identities.identifier, identities.name, identities.metadata
+  FROM identities
+  WHERE ( identities.auth_method = ? )
+  ORDER BY identities.auth_method, identities.identifier
+`)
+
+var identityObjectsByAuthMethodAndType = RegisterStmt(`
+SELECT identities.id, identities.auth_method, identities.type, identities.identifier, identities.name, identities.metadata
+  FROM identities
+  WHERE ( identities.auth_method = ? AND identities.type = ? )
+  ORDER BY identities.auth_method, identities.identifier
+`)
+
+var identityObjectsByAuthMethodAndIdentifier = RegisterStmt(`
+SELECT identities.id, identities.auth_method, identities.type, identities.identifier, identities.name, identities.metadata
+  FROM identities
+  WHERE ( identities.auth_method = ? AND identities.identifier = ? )
+  ORDER BY identities.auth_method, identities.identifier
+`)
+
+var identityObjectsByAuthMethodAndName = RegisterStmt(`
+SELECT identities.id, identities.auth_method, identities.type, identities.identifier, identities.name, identities.metadata
+  FROM identities
+  WHERE ( identities.auth_method = ? AND identities.name = ? )
+  ORDER BY identities.auth_method, identities.identifier
+`)
+
+var identityObjectsByType = RegisterStmt(`
+SELECT identities.id, identities.auth_method, identities.type, identities.identifier, identities.name, identities.metadata
+  FROM identities
+  WHERE ( identities.type = ? )
+  ORDER BY identities.auth_method, identities.identifier
+`)
+
+var identityID = RegisterStmt(`
+SELECT identities.id FROM identities
+  WHERE identities.auth_method = ? AND identities.identifier = ?
+`)
+
+var identityCreate = RegisterStmt(`
+INSERT INTO identities (auth_method, type, identifier, name, metadata)
+  VALUES (?, ?, ?, ?, ?)
+`)
+
+var identityDeleteByAuthMethodAndIdentifier = RegisterStmt(`
+DELETE FROM identities WHERE auth_method = ? AND identifier = ?
+`)
+
+var identityDeleteByNameAndType = RegisterStmt(`
+DELETE FROM identities WHERE name = ? AND type = ?
+`)
+
+var identityUpdate = RegisterStmt(`
+UPDATE identities
+  SET auth_method = ?, type = ?, identifier = ?, name = ?, metadata = ?
+ WHERE id = ?
+`)
+
+// identityColumns returns a string of column names to be used with a SELECT statement for the entity.
+// Use this function when building statements to retrieve database entries matching the Identity entity.
+func identityColumns() string {
+	return "identity.id, identity.auth_method, identity.type, identity.identifier, identity.name, identity.metadata"
+}
+
+// getIdentitys can be used to run handwritten sql.Stmts to return a slice of objects.
+func getIdentitys(ctx context.Context, stmt *sql.Stmt, args ...any) ([]Identity, error) {
```

Comment:
@masnax do you know of any way to avoid this incorrect pluralisation when using `lxd-generate`?

Reply:
@tomponline are you happy for me to fix this retrospectively?

Reply:
Yep

Reply:
I've created issue #12815 and assigned myself to it to implement later.